<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpektrLaden</class>
 <widget class="QMainWindow" name="SpektrLaden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spektroskopie - Messung einlesen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>947</width>
      <height>425</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticallayout">
       <item>
        <layout class="QHBoxLayout" name="horizontallayout2">
         <item>
          <widget class="QLineEdit" name="edit_pfad">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_aendern">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Ã„ndern...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalspacer0">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="button_konfig">
         <property name="text">
          <string>Messkonfiguration aus Datei einlesen</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout1">
         <item>
          <layout class="QGridLayout" name="gridlayout0">
           <item row="0" column="1">
            <widget class="QSpinBox" name="box_df">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_mittelungen">
             <property name="text">
              <string>Anzahl Mittelungen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="box_mittelungen">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_df">
             <property name="text">
              <string>delta f (Hz)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalspacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridlayout1">
           <item row="0" column="0">
            <widget class="QLabel" name="label_fmin">
             <property name="text">
              <string>Freq Min (kHz)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_fmax">
             <property name="text">
              <string>Freq Max (kHz)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="box_fmin">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="box_fmax">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>0.500000000000000</double>
             </property>
             <property name="maximum">
              <double>999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalspacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalspacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout4">
         <item>
          <widget class="QLabel" name="label_methode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Lorentzverteilung</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="box_methode">
           <item>
            <property name="text">
             <string>Antreibendes System</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cantilever</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Phase</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_bereich">
         <property name="text">
          <string>Frequenzbereich begrenzen (in Messpunkten):</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout3">
         <item>
          <widget class="QLabel" name="label_bereich_links">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>links:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="box_bereich_links">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_bereich_rechts">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>rechts:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="box_bereich_rechts">
           <property name="minimum">
            <number>-999999999</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalspacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_fitparameter">
         <property name="text">
          <string>Fitparameter:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout5">
         <item>
          <widget class="QLabel" name="label_amp_min">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Amplitude von</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_amp_min">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="value">
            <double>0.000010000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_amp_max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>bis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_amp_max">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout7">
         <item>
          <widget class="QLabel" name="label_untergrund_min">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Untergrund von</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_untergrund_min">
           <property name="decimals">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_untergrund_max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>bis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_untergrund_max">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="value">
            <double>0.005000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout6">
         <item>
          <widget class="QLabel" name="label_guete">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>GÃ¼te =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_guete">
           <property name="value">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_guete_min">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>von</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_guete_min">
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_guete_max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>bis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_guete_max">
           <property name="value">
            <double>25.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalspacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout0">
         <item>
          <widget class="QPushButton" name="button_fitten">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Fitten</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progress_bar">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalspacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticallayout1">
       <item>
        <widget class="Vorschau" name="plotter" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontallayout8">
         <item>
          <widget class="QPushButton" name="button_vorschau">
           <property name="text">
            <string>Fitvorschau anzeigen</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalspacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_omega">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Omega:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="box_omega">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ac">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>AC (V):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_ac">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-99.999999000000003</double>
           </property>
           <property name="maximum">
            <double>99.999999000000003</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_dc">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>DC (V):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_dc">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-99.999999000000003</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vorschau</class>
   <extends>QWidget</extends>
   <header>Module.Spektroskopie.Vorschau</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_vorschau</sender>
   <signal>toggled(bool)</signal>
   <receiver>box_omega</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_vorschau</sender>
   <signal>toggled(bool)</signal>
   <receiver>box_ac</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_vorschau</sender>
   <signal>toggled(bool)</signal>
   <receiver>box_dc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>900</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_vorschau</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_omega</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_vorschau</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_ac</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_vorschau</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_dc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
